website/README.md

```diff
@@ -6,21 +6,21 @@ We recommend using `nvm` to manage npm versions. See [nvm installation instructi
 
 With `nvm` installed, run the following command from within the `website` directory:
 
-```shell
+```shell copy
```

Comment:
I don't think we need this here (or any other MD file that's not rendered by our build)